gigadb/app/tools/transfer-files/config-sources/rclone.conf.dist

```diff
@@ -0,0 +1,18 @@
+[wasabi]
```

Comment:
**question**: We do not need this template file because there are no variable values to interpolate in order to create `rclone.conf`?

Reply:
**answer:** The dist file and its related steps have been removed in https://github.com/gigascience/gigadb-website/pull/1977/commits/f0328c5a596b1b6e663e4a920159b00972b018d0